src/early_multicellular_stage/EarlyMulticellularSpecies.cs

```diff
@@ -79,8 +79,7 @@ public override void UpdateInitialCompounds()
                 continue;
 
             // Initial compounds should suffice for a fixed amount of time.
-            var compoundInitialAmount = (Math.Abs(compoundBalance.Value.Balance) / Cells.Count) *
-                Constants.INITIAL_COMPOUND_TIME;
+            var compoundInitialAmount = Math.Abs(compoundBalance.Value.Balance) * Constants.INITIAL_COMPOUND_TIME;
```

Comment:
Could you add another constant multiplier here to tweak the single cell value for a colony (1.5 I think would be fine)? Just to get a bit closer to what the behaviour was before and ensure that spawned colonies of the player's species don't die super fast.